PerfTools/AllocMonitor/scripts/edmModuleAllocJsonToCircles.py

```diff
@@ -0,0 +1,387 @@
+#!/usr/bin/env python3
+import json
+transitionTypes = ["construction", "event",
+                        "begin job", "begin stream",
+                        "global begin run", "global begin luminosity block",
+                        "stream begin run", "stream begin luminosity block",
+                        ]
+
+def processModuleTransition(moduleLabel, moduleType, moduleInfo, transitionType, moduleTransition):
+        moduleTransition[moduleLabel] = {"cpptype": moduleType, "allocs": [], "nEvents": 0}
+        for entry in moduleInfo:
+            if entry["transition"] == transitionType:
+                moduleTransition[moduleLabel]["allocs"].append(entry.get("alloc",{}))
+        moduleTransition[moduleLabel]["nEvents"] = len(moduleTransition[moduleLabel]["allocs"])
```

Comment:
This is where nEvents is set from the length of allocs.

Reply:
Using this for nEvents has proven to yield some unexpected values because some modules have more than nEvent allocs.

Reply:
For example  99 "events" for GlobalTag in step4.

Reply:
Or 1760 "events" for source in step2.

Reply:
The number of transitions is used to get the average.

Reply:
The is another field which contains the event number. Maybe I should use that for event count.

I believe the circles javascript use the event count to determine the average per event. I need to check.